README.md

```diff
@@ -69,7 +69,7 @@ Please use the same value as `operator_private_key` in your L1 + L2 config files
 ## Ether.fi Admin Flow
```

Comment:
add a link to the table above this comment. I can't mark the section specifically because there were no changes there

Reply:
added